projects/media-viewer/src/lib/annotations/comment-set/comment-set.component.ts

```diff
@@ -22,6 +25,8 @@ export class CommentSetComponent implements OnInit, OnDestroy {
   @Input() rotate: number;
   @Input() height: number;
 
+  @Output() unsavedChanges = new EventEmitter<boolean>();
```

Comment:
Wouldn't a subject be a more angular way?

Reply:
with a service? otherwise how would that pass information outwards?